Dockerfile

```diff
@@ -666,17 +666,11 @@ RUN \
   echo "$X265_SHA256  x265.tar.bz2" | sha256sum --status -c - && \
   tar xf x265.tar.bz2 && \
   cd x265_*/build/linux && \
-  cmake \
-    -G"Unix Makefiles" \
-    -DCMAKE_VERBOSE_MAKEFILE=ON \
-    -DENABLE_SHARED=OFF \
-    -DENABLE_AGGRESSIVE_CHECKS=ON \
-    -DCMAKE_ASM_NASM_FLAGS=-w-macro-params-legacy \
-    -DENABLE_CLI=OFF \
-    -DENABLE_NASM=ON \
-    -DCMAKE_BUILD_TYPE=Release \
-    ../../source && \
-  make -j$(nproc) install
+  sed -i '/^cmake / s/$/ -DCMAKE_VERBOSE_MAKEFILE=ON -DENABLE_AGGRESSIVE_CHECKS=ON -DCMAKE_ASM_NASM_FLAGS=-w-macro-params-legacy -DENABLE_NASM=ON -DCMAKE_BUILD_TYPE=Release/' ./multilib.sh && \
+  sed -i 's/ -DENABLE_SHARED=OFF//g' ./multilib.sh && \
+  sed -i '/^cmake / s/$/ -DENABLE_SHARED=OFF/' ./multilib.sh && \
```

Comment:
My sed skills is not what they should be. This add flags last to cmake lines and then moves `-DENABLE_SHARED=OFF` last?

Wonder if it is possible to pass arguments or defines via env? another alternative  is to use sed to add a `${CMAKEFLAGS}` similar to MAKEFLAGS and use that?